includes/Hooks.php

```diff
@@ -456,4 +456,21 @@ private static function formatTagError( Parser $parser, array $msg ) {
 			'</strong>';
 	}
 
+    /**
+     * @param $out
+     * @param $text
+     * @return true
+     */
+    public static function onOutputPageBeforeHTML( &$out, &$text ) {
```

Comment:
Return early if the `OutputPage` object isn't under a `Template` namespace and isn't a `.css` file

Reply:
After convo with @akiel15 this doesn't make sense, as the HTML content has to be changed on every page where TemplateStyles is used